### PR TITLE
Vet Clinic

### PR DESCRIPTION
**Pull request by altontonn**
# Vet Clinic SQL 💎 

I added SQL for the following files:-

- Creating table 👉 `schema.sql`
- Inserting to the table 👉 `data.sql`
- Data selection 👉 `queries.sql`
<h3>Screenshots</h3>

1. List the date of birth of all animals named either "Agumon" or "Pikachu"
 two-names

2. Find all animals whose name ends in "mon".
 mon

3. List the name of all animals born between 2016 and 2019.
 birth

4. List the name of all animals that are neutered and have less than 3 escape attempts.
 neutered
 
5. List the name and escape attempts of animals that weigh more than 10.5kg
KG

6. Find all animals that are neutered.
 neutered

7. Find all animals not named Gabumon.
Exclude

8. Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg) 
precise

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,9 @@
 /*Queries that provide answers to the questions from all projects.*/
-
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM ANIMALS WHERE name LIKE '%mon%';
+SELECT NAME FROM ANIMALS WHERE DATE_OF_BIRTH BETWEEN '2016-01-01' AND '2019-01-01';
```
Comment (Mihndim2020) [inline queries.sql]: - [OPTIONAL] The range should is between the beggining of 2016 and end of 2019, something like so: 
`SELECT name FROM animals WHERE date_of_birth >= '2016/01/01' AND date_of_birth <= '2019/31/12'` please kindly fix that.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @altontonn, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

Please also consider:
- [ ] As it is a [requirement ](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_create_tables.md)for this project, " `Remember to add the screenshot with the results of your queries to your Pull Request description.` ", please kindly add screenshots of your queries in you pull request summary. 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @altontonn, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 


 ## Required Changes ♻️
- [ ] [REQUIRED] Good job so far 👍 You put screenshots of your queries results in your project but you didn't put them in the `Pull Request description`, they should be visible in the `Pull Request` description. Please add them to the `PR` description 😃 

image

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShahierNashaat — APPROVED** [review]
Hi @altontonn, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Your code is clean and well documented ✔️

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**